.travis.yml

```diff
@@ -2,23 +2,41 @@ language: node_js
 node_js:
   - "6"
 
+stages:
+  - lint_build
+  - test
+
 env:
   global:
     - LAUNCHER=Node
 
-matrix:
+jobs:
   include:
-    - env: LAUNCHER=Firefox
-    - env: LAUNCHER=Chrome
-    - node_js: "9"
-    - node_js: "8"
-    - node_js: "6"
-    - node_js: "4"
+    - stage: lint_build
+      before_install: skip
+      install: npm install uglify-js@3.1.x eslint@4.7.x
```

Comment:
Doing these specific package installs will make things harder to maintain. If you ever change a pacakge version, you need to change it in multiple places.

I think we should leave `npm install` to run completely.
To help improve `npm install` speed We can cache node_modules folder, as per these instructions:
https://docs.travis-ci.com/user/caching
https://blog.travis-ci.com/2013-12-05-speed-up-your-builds-cache-your-dependencies

Travis also supports yarn now, which might add some speed improvement:
https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn